include/vkd3d_shader.h

```diff
@@ -57,8 +57,8 @@ typedef uint64_t vkd3d_shader_hash_t;
 struct vkd3d_shader_meta
 {
     vkd3d_shader_hash_t hash;
-    uint32_t cs_workgroup_size[3]; /* Only contains valid data if uses_subgroup_size is true. */
-    uint32_t patch_vertex_count; /* Relevant for HS. May be 0, in which case the patch vertex count is not known. */
+    unsigned int cs_workgroup_size[3]; /* Only contains valid data if uses_subgroup_size is true. */
```

Comment:
Why did this change? These are equivalent.

Reply:
The DXIL C API takes these by pointers, and uint32_t might not be == unsigned int.

Reply:
I'll reorder the changes so they make more sense.